src/redux/peopleSlice.js

```diff
@@ -0,0 +1,26 @@
+import {createSlice} from '@reduxjs/toolkit';
+
+const initialState = {
+  dataLoadState: 0, // 0 - not loaded, 1 - is loading, 2 - loaded, 3 - error
+  dataLoadError: null,
+  data: [],
+};
```

Comment:
Change state structure a bit:
`
{
  isLoading: false,
  error: null,
  data: [],
}
`

Reply:
Do not make loading state too complex. It should be boolean type, without 0, 1, 2, 3 statuses